src/modules/ekf2/ekf2_main.cpp

```diff
@@ -691,6 +691,14 @@ void Ekf2::run()
 	vehicle_status_s vehicle_status = {};
 	sensor_selection_s sensor_selection = {};
 
+	// get heading offset required for use of GPS yaw
+	param_t handle = param_find("GPS_YAW_OFFSET");
```

Comment:
Add this to vehicle_gps_position instead of accessing the parameter directly?

Reply:
We could do, but that will increase log size unnecessarily

Reply:
Are you able to explain the reasons for not wanting to adopt the approach used here of  checking the  parameter value on startup? It seems like the simplest way to achieve the required result.

Reply:
Generally it ends up being a bad idea to cross module boundaries. These parameters are internal data of that driver or module, not a system wide setting.

In the simple intended use case I'm sure this is fine, but how does it scale when there are multiple GPS receivers, or GPS from UAVCAN, Mavlink, or some mix of all of the above. What about the GPS being connected to one system, but running the estimator on another with messages passed via FastRTPS?

Yes I'm jumping ahead, but these are the reasons to try and keep things as modular as possible as we go along .

Reply:
@dagar Can you show me an example of a 'low rate (nearly static) metadata message'